samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md

```diff
@@ -43,3 +36,19 @@ To run the sample, follow these steps:
 6. Update the sample with Event Hubs namespace and Storage account name.
 7. Run [Sender application](https://github.com/Azure/azure-event-hubs/tree/serkar.AddCustomRbacEhSample/samples/DotNet/Microsoft.Azure.EventHubs/SampleSender) to send some number of messages to your event hub.
 8. Run the CustomRole sample to receive those events back.
+
+**Note:** This sample uses the legacy Event Hubs library `Microsoft.Azure.EventHubs`. We strongly recommend you to use the new library `Azure.Messaging.EventHubs`. 
+
+See [this](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/samples/DefiningCustomCredentialTypes.md#authenticating-with-the-on-behalf-of-flow) sample that uses the new library to authenticate after the environment variables for client id and secret are set. 
```

Comment:
Avoid using `this` or `here`  or `link` as link text. This hinders accessibility for screenreader users when they try to go over all links on the page. When doing so, link texts are read out loud. Hearing "this" a bunch of times gives no context as to what the link would be